docs/guides/getting-started.rst

```diff
@@ -3,8 +3,7 @@ Getting Started with DebOps
 
 .. include:: ../includes/all_from_subdir.rst
 
-Welcome to DebOps! You have installed Ansible and DebOps scripts, downloaded
-the roles and playbooks, and are wondering where to go next? Here you can read
+Welcome to DebOps! You have installed `Ansible <https://docs.debops.org/en/latest/debops/docs/installation.html#debops-prerequisites>`_ and `DebOps scripts <https://docs.debops.org/en/latest/debops/docs/index.html>`_, downloaded the roles and `playbooks <https://docs.debops.org/en/latest/debops-playbooks/docs/index.html>`_, and are wondering where to go next? Here you can read
```

Comment:
True :) That is what the `.. include:: ../includes/all_from_subdir.rst` is about :)